source/element-parameters.md

```diff
@@ -53,11 +62,15 @@ Element:
   inherit: q2
 ```
 
----
+%---------------------------------------------------------------------------------------------------
+(s:ac.kicker.params)=
+## ACKickerP  AC-Kicker Parameters
+
+In Construction...
 
 %---------------------------------------------------------------------------------------------------
-(s:aperture.group)=
-## ApertureP Parameter Group
+(s:aperture.params)=
+## ApertureP Aperture Parameters
 
```

Comment:
The convention of appending `P` to an element type in order to produce an associated parameter group is rather confusing.  What about being explicit, again?  For example, just call this `Aperture_Parameters`?

Reply:
I have added a sentence to explain this convention.